src/AppKit/NSPrintInfo.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using XamCore.PrintCore;
+
+namespace XamCore.AppKit {
+	public partial class NSPrintInfo {
+		public PMPrintSession PMPrintSession {
```

Comment:
The reason I didn't drop the PM prefix is because PrintSettings already exists as a separate property and I left the other 2 so they would all use the same format.  Per our discussion on slack, changing them to getter methods instead of properties, so GetPrintSession (), etc.